torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -157,8 +157,16 @@ IValue toIValue(py::handle obj, const TypePtr& type, c10::optional<int32_t> N) {
               attrName));
         }
 
-        const auto& contained = py::getattr(obj, attrName.c_str());
-        userObj->setSlot(slot, toIValue(contained, attrType));
+        try {
+          const auto& contained = py::getattr(obj, attrName.c_str());
+          userObj->setSlot(slot, toIValue(contained, attrType));
+        } catch (std::exception& e) {
+          throw py::cast_error(c10::str(
+              "Could not cast attribute ",
+              attrName,
```

Comment:
Nit:
Surround `attrName` with single quotes to make it clearer that it is a name of attribute.

Reply:
Done.